webpack-test/configCases/loaders/issue-3320/index.js

```diff
@@ -1,4 +1,5 @@
-it("should resolve aliased loader module with query", function() {
+// Webpack supports this, but Rspack does not support `resolve.alias` with a custom query and fragment
+it.skip("should resolve aliased loader module with query", function() {
```

Comment:
Currently, Due to the fact that nodejs-resolver does not support resolving with alias, Rspack does not support resolve aliased loader. We should fix this.

Reply:
tracked here https://github.com/web-infra-dev/rspack/issues/4009